batch/src/main/scala/uk/gov/ons/addressindex/Main.scala

```diff
@@ -95,6 +95,7 @@ For usage see below:
         } else {
           Mappings.hybrid
         })
+      .param("include_type_name", "true")
```

Comment:
Why do we need this? Deprecated in 7.0.0

Reply:
See comments in https://collaborate2.ons.gov.uk/jira/browse/FWMT-1317

Reply:
Do we have a task in the backlog to fix this? I see that this "fix" will not work in 8.x so we're just putting it off. Tech Debt and all that.

Reply:
To avoid the deprecation the mappings would have to be rejigged in a big way, so in the interests of not taking on too much risk I would rather get it working with the current mappings and upgrade them later with a separate ticket.

Reply:
Ok. But I think that task needs to be created and added to the backlog.

Reply:
Ticket created https://collaborate2.ons.gov.uk/jira/browse/FWMT-1655